readthedocs/search/tests/test_api.py

```diff
@@ -158,6 +158,8 @@ def test_doc_search_filter_by_version(self, api_client, project):
         assert len(data) == 1
         assert data[0]['project'] == project.slug
 
+    @pytest.mark.skip('To be investigated')
```

Comment:
Forse non abbiamo uno _standard_ per questo genere di cose, ma potremmo lasciare un commento tipo `#TODO` per le cose che devono essere riviste in futuro?

Reply:
hai ragione, mi sono basato solo sul mark di `pytest` che però potrebbe non essere ovvio
ho aggiunto dei commenti `FIXME`